pkg/providers/launchtemplate/nodeclass_test.go

```diff
@@ -742,7 +749,10 @@ var _ = Describe("EC2NodeClass/LaunchTemplates", func() {
 		It("should calculate memory overhead based on eni limited pods when ENI limited", func() {
 			ctx = settings.ToContext(ctx, test.Settings(test.SettingOptions{
```

Comment:
Do we need to explicitly set this in the settings if this is the default?

Reply:
We don't need to, but one could argue it makes the test more readable. I could go either way on dropping it.